packages/create-sitecore-jss/src/init-runner.ts

```diff
@@ -41,16 +41,19 @@ export const initRunner = async (initializers: string[], args: BaseArgs) => {
 
   await runner(initializers);
 
-  saveConfiguration(args.templates, path.resolve(`${args.destination}${sep}package.json`));
-  // final steps (install, lint, etc)
-  if (!args.noInstall) {
-    installPackages(args.destination, args.silent);
-    lintFix(args.destination, args.silent);
-  }
+  for (const destination of [args.destination, args.nodeAppDestination]) {
+    if (!destination) continue;
+    saveConfiguration(args.templates, path.resolve(`${destination}${sep}package.json`));
```

Comment:
Not sure if saving the applied "templates" makes sense for the proxy app. Should we save this saveConfiguration operation only for the primary app?